pages/02.Tutorials/docs.en.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Tutorials
+page-toc:
+    active: false
+taxonomy:
+    category:
+        - tutorials
```

Comment:
```suggestion
        - docs
```
All other taxonomy category values are "docs" too.
Or rename the category of the german variant to "tutorials" instead.